src/Repository/ConferenceRepository.php

```diff
@@ -53,9 +53,9 @@ public function findAttendedConferences(): array
     public function findAttendedConferencesByTag(string $tag): array
     {
         return $this->createAttendedQueryBuilder()
-            ->leftJoin('c.tags', 't')
-            ->andWhere('t.name = :tagName')
-            ->setParameter('tagName', $tag)
+            ->select('c')
+            ->andWhere('c.tags LIKE :tagName')
+            ->setParameter('tagName', '%"'.$tag.'"%')
```

Comment:
😬 Let's use a real SQL function for that. something like `JSON_CONTAINS()`

There are some doctrine extension OSS. Copy what you need

Reply:
If I remember well, there's already the needed extension. I has to use it for the Workflow part, to retrieve if an object had one of the given states (stored in JSON)